agent_factory/scaffold/task_fetcher.py

```
@@ -0,0 +1,188 @@
+            max_tasks: Maximum tasks to return
+            labels: Optional label filter (e.g., ["build", "rivet-pro"])
+
+        Returns:
+            List of task dicts, sorted by priority (highest first)
+        """
+        # Check cache
+        if self._cache and (time.time() - self._cache_time < self._cache_ttl):
+            logger.debug("Using cached tasks")
+            filtered = self._cache
+            if labels:
+                filtered = [t for t in filtered if any(l in t.get("labels", []) for l in labels)]
```

Ruff (0.14.8)

Ambiguous variable name: `l`

(E741)